server/Dockerfile

```diff
@@ -2,19 +2,23 @@ FROM node:14.14.0-alpine
 
 WORKDIR /app
 
-RUN apk add postgresql-dev
+RUN apk add postgresql-dev python-dev build-base
```

Comment:
Comment for this? Is this something required by pg types for typescript?

Reply:
I've confirmed that this isn't necessary in the other PR, so maybe we could remove? :)

Reply:
This _may_ be necessary for the live code reloading stuff, but I would be a bit surprised frankly, since I thought it was a pure-js thing in use. Agree that we should remove if unneeded. It may also have been for debugging purposes.